client-src/default/webpack.config.js

```diff
@@ -15,4 +17,12 @@ module.exports = {
       },
     ],
   },
+  plugins: [
+    new webpack.NormalModuleReplacementPlugin(/\/clients\//, (resource) => {
```

Comment:
memo: We will delete this plugin at the next major version because we can change the directory structure.

Reply:
We don't need this plugin what potentially break other applications, just create is file and re-require logic